src/components/ResultsListItem.vue

```diff
@@ -27,11 +29,13 @@ export default {
   methods: {
     savePark() {
       this.$store.commit('savePark', this.result)
-    },
-    unsavePark() {
-      // implement code to unsave parks here
     }
   },
+  computed: {
+    saved() {
```

Comment:
These `computed` properties are so versatile. I've been really enjoying learning and utilizing this awesome Vue feature.

Reply:
Yeah, I think this has been my favorite part of Vue so far.  There is so much power embedded in the `computed properties`!